<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPEManifest</class>
 <widget class="QDialog" name="DialogPEManifest">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Manifest Info</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabInfo">
      <attribute name="title">
       <string>Info</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextEditX" name="textEditInfo">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSource">
      <attribute name="title">
       <string>Source</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEditX" name="textEditManifest">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonOK">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPushButtonX</class>
   <extends>QPushButton</extends>
   <header>qpushbuttonx.h</header>
  </customwidget>
  <customwidget>
   <class>QTextEditX</class>
   <extends>QTextEdit</extends>
   <header>qtexteditx.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
